qanary_pipeline-template/src/main/java/eu/wdaqua/qanary/PipelineExplanationHelper.java

```diff
@@ -0,0 +1,104 @@
+package eu.wdaqua.qanary;
```

Comment:
@dschiese Please add a documentation to this class and its methods.

Reply:
Done with https://github.com/WDAqua/Qanary/pull/293/commits/0a69fca7c3ddf765903eb565e0664c37f822dea6